src/supporting-files/stackbit-banner.html.js

```diff
@@ -7,7 +7,7 @@ module.exports = `<div id="theme-bar" class="theme-bar theme-bar-fixed theme-bar
   <div class="theme-bar-center">
     {% if stackbit_banner.github_url %}
     <a
-      class="theme-bar-button theme-bar-button-outlined theme-bar-fork"
```

Comment:
`theme-bar-fork` class was added by me - as far as I remember, it can be removed